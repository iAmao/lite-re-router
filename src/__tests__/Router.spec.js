import React from 'react';

import renderer from 'react-test-renderer';

import { shallow } from 'enzyme';

import { Router } from '../';


describe('Router', () => {
  describe('Snapshot Test', () => {
    it('should render the router component', () => {
      const tree = renderer.create(<Router />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
