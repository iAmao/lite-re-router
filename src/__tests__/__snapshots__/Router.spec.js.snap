// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router Snapshot Test should render the router component 1`] = `
<div>
  <h3>
    Routing
  </h3>
</div>
`;
